.travis.yml

```diff
@@ -1,50 +0,0 @@
-os: osx
-osx_image: xcode10.2
-rvm: 2.6.0
-
-addons:
-  homebrew:
-    packages:
-    - swiftlint
-    update: true
-
-cache:
-  - bundler
-  - cocoapods
-
-branches:
-  only:
-    - master
-
-env:
-   global:
-     - WORKSPACE="ReactiveLists.xcworkspace"
-     - FRAMEWORK_SCHEME="ReactiveLists"
-     - EXAMPLE_SCHEME="Example"
-     - SDK=iphonesimulator
-
-   matrix:
-     - DEST="OS=11.4,name=iPhone X" POD_LINT="YES"
-     - DEST="OS=12.2,name=iPhone X" POD_LINT="NO"
-
-before_install:
-  - gem update --system
-  - gem install bundler
-  - gem install cocoapods --pre
-
-script:
-  - set -o pipefail
-  - xcodebuild -version -sdk
-
-  - if [ $POD_LINT == "YES" ]; then
-        bundle exec pod lib lint;
-    fi
-
-  - xcodebuild build -workspace "$WORKSPACE" -scheme "$EXAMPLE_SCHEME" -sdk "$SDK" -destination "$DEST" ONLY_ACTIVE_ARCH=NO | bundle exec xcpretty -c;
```

Comment:
We are missing an equivalent step that makes sure the Example app continues to build. Can we add that too?

Reply:
Oh yeah def